3.md

```diff
@@ -0,0 +1,4 @@
+Как зарегистрированный пользователь,  
```

Comment:
Does "have access" mean "be able to get a list of all documents"?

Reply:
yes(fixed)